src/qt/transactiondesc.cpp

```diff
@@ -167,9 +146,6 @@ QString TransactionDesc::toHTML(CWallet *wallet, CWalletTx &wtx, TransactionReco
 
         if (fAllFromMe)
         {
-            if(fAllFromMe & ISMINE_WATCH_ONLY)
-                strHTML += "<b>" + tr("From") + ":</b> " + tr("watch-only") + "<br>";
```

Comment:
Yes, but word it differently please. There is no need to display this as `From: watch-only`. But it needs to be visible that the transaction is watch-only.